guides/src/getting-started/getting-started.md

```diff
@@ -54,7 +54,7 @@ Type the following Ruby code in your editor, and save it as `lib/hello.rb`.
 currencies = { US: "$", JP: "¥", UK: "£" }
 country = %w(US JP UK).sample()
 
-puts "Hello! The price is #{currencies[country.to_sym]}100. 💸"
```

Comment:
The mistake in question

Reply:
`country` is a `String` (or `nil`), not `Symbol`.